commissioning.tex

```diff
@@ -1,27 +1,48 @@
 \section{Rubin Observatory Commissioning}
 \label{sec:commissioning}
 
+\subsection{Commissioning Milestones}
+\label{ssec:commissioning-milestones}
+
+Commissioning work is being planned around two major milestones, ``First Photon'' and ``System First Light''. 
+
+\textbf {First Photon}: Defined as the very first photons to pass through the optical system and fall on the  focal plane to produce an image of the night sky.
+
+\textbf {System First Light}: Defined as the point at which we can routinely acquire science-grade imaging across the full focal plane (LSSTCam) and have a well understood technical path towards meeting the Construction Completeness criteria   \citeds{sitcomtn-061}.
+Also referred to as \textbf{First Light}. 
+These images will be shared with the worldwide community. 
+
+First Photon occurs following the successful completion of system integration. 
+There are no quality criteria applied to the images produced for the First Photon milestone. 
+First Photon  marks the start of the the on-sky engineering phase.
+System First Light  marks the end of the the on-sky engineering phase and the start of the System Optimization and Science Validation phases of commissioning.
+The time from First Photon to System First Light is anticipated to be three months.
```

Comment:
```suggestion
The time from First Photon to System First Light is anticipated to be 1.5 months.
```

Reply:
Removed - we now have > 1 "First Photon"